tests/unit/modules/test_file.py

```diff
@@ -907,9 +907,9 @@ def test_extract_hash(self):
         with tempfile.NamedTemporaryFile(mode='w+b', delete=False) as tfile:
             tfile.write(salt.utils.stringutils.to_bytes(
                 'rc.conf ef6e82e4006dee563d98ada2a2a80a27\n'
-                'ead48423703509d37c4a90e6a0d53e143b6fc268 example.tar.gz\n'
-                'fe05bcdcdc4928012781a5f1a2a77cbb5398e106 ./subdir/example.tar.gz\n'
-                'ad782ecdac770fc6eb9a62e44f90873fb97fb26b foo.tar.bz2\n'
+                'ead48423703509d37c4a90e6a0d53e143b6fc268  example.tar.gz\n'
+                'fe05bcdcdc4928012781a5f1a2a77cbb5398e106  ./subdir/example.tar.gz\n'
```

Comment:
Is there a reason why there are 2 spaces after the hash (instead of the original 1 space) for these 2 items? If not, would you change these 2 back?

Reply:
That’s the output I get from shasums.

Reply:
Could you test at least one of those two with just one space, to ensure backwards compatibility is preserved?